app/cardrecord.js

```diff
@@ -88,6 +36,49 @@ class CardRecorder extends MyTrello {
         return deferred.promise;
     }
 
+    cardRecordFunctions(card){
```

Comment:
Ticky comment: this function name is throwing me for a loop.  Does this name describe its functionality?